bin/propolis-server/src/lib/stats/virtual_machine.rs

```diff
@@ -495,10 +495,9 @@ mod test {
                 )
                 .unwrap_or_else(|| {
                     panic!(
-                        "kstat state '{}' did not map to an oximeter state, \
+                        "kstat state '{kstat_state}' did not map to an oximeter state, \
```

Comment:
this crests well past 80 columns but i don't think rustfmt knows how to reflow strings like this (sorry)